ArduCopter/mode_auto.cpp

```diff
@@ -600,6 +600,9 @@ bool ModeAuto::start_command(const AP_Mission::Mission_Command& cmd)
 
     case MAV_CMD_NAV_WAYPOINT:                  // 16  Navigate to Waypoint
         do_nav_wp(cmd);
+        wp_nav->set_wp_radius(float(cmd.p1 * 100));
+    
+        gcs().send_text(MAV_SEVERITY_CRITICAL,"Wavpoint_radius = %d",int(wp_nav->get_wp_radius_cm()/100));
```

Comment:
I think this message is at the notification or information level.

Reply:
Yes this is information level message which is just for debugging purpose to exactly know WP_radius at that waypoint